src/server/migrations/20200325162239_crops.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('crops', function(table) {
+    table.increments();
```

Comment:
It should be 
 table
      .increments('id')
      .primary()
  .notNullable();

Reply:
I can alter it again but when we use .increments then we dont need to right 'id' when it runs it just add an id column automatically :)

Reply:
Good to know Amjad.
But can it also add primary key, excited to know?

Reply:
Yes sure, it will be a primary key as well.

Reply:
That should work fine yes 👍